plugins/Xpressive/Xpressive.cpp

```diff
@@ -252,7 +250,7 @@ void Xpressive::smooth(float smoothness,const graphModel * in,graphModel * out)
 		const int guass_size = (int)(smoothness * 5) | 1;
 		const int guass_center = guass_size/2;
 		const float delta = smoothness;
-		const float a = 1.0f / (std::sqrt(numbers::tau_v<float>) * delta);
+		const float a = 1.f / (std::sqrt(2 * std::numbers::pi_v<float>) * delta);
```

Comment:
```suggestion
		const float a = std::numbers::inv_sqrtpi_v<float> / (std::numbers::sqrt2_v<float> * delta);
```

$$\frac{1}{\sqrt{2\pi}\cdot delta}=\frac{1}{\sqrt{\pi}}\cdot\frac{1}{\sqrt{2}\cdot delta}$$